pkg/api/store/globalrolebindings/globalrolebinding_store.go

```diff
@@ -8,21 +8,48 @@ import (
 
 const GrbVersion = "field.cattle.io/grbUpgrade"
 
-func Wrap(store types.Store, grbLister v3.GlobalRoleBindingLister) types.Store {
+func Wrap(store types.Store, grLister v3.GlobalRoleLister) types.Store {
 	return &grbStore{
-		Store:     store,
-		grbLister: grbLister,
+		Store:    store,
+		grLister: grLister,
 	}
 }
 
 type grbStore struct {
 	types.Store
-	grbLister v3.GlobalRoleBindingLister
+
+	grLister v3.GlobalRoleLister
 }
 
 func (s *grbStore) Create(apiContext *types.APIContext, schema *types.Schema, data map[string]interface{}) (map[string]interface{}, error) {
 
 	values.PutValue(data, "true", "metadata", "annotations", GrbVersion)
 
+	err := s.addOwnerReference(data)
+	if err != nil {
```

Comment:
The error should be checked for not exists and an error code returned accordingly